### PR TITLE
setup graphene

### PR DESCRIPTION
**Pull request by bluquar**
new PR to replace #16 since I can't figure out how to get that one to stop showing already-committed changes

Screen Shot 2021-11-20 at 10 28 09 AM

### CODE REVIEW

**Review by lowellbander — DISMISSED** [review]

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+GRAPHQL_HOST_URL="http://localhost:8000/graphql"
+
+DJANGO_SECRET_KEY="django-insecure-iya5ffv=5l5^s2ajur!p-#!_zb7opbp18z9oxruz75a#i8*b^d"
+DJANGO_DEBUG="True"
+
+REACT_NATIVE_WEB_CLIENT_DOMAIN="http://localhost:19006"
```
Comment (lowellbander) [inline .env:6]: note: this should def not be here

File: App.tsx [hunk 2]
```diff
@@ -9,6 +9,14 @@ import useCachedResources from './hooks/useCachedResources';
 import useColorScheme from './hooks/useColorScheme';
 import Navigation from './navigation';
 
+import {ApolloClient, InMemoryCache, ApolloProvider} from '@apollo/client';
+
+// Initialize Apollo Client
+const client = new ApolloClient({
+  uri: 'http://localhost:8000/graphql',
```
Comment (lowellbander) [inline App.tsx:16]: wanna use the env here?

**Review by lowellbander — APPROVED** [review]
lit, tho looks like there are some stale changes in here (e.g. not using env in that react file) and also weird that .env is tracked even though it's in .gitignore

File: App.tsx (hunk 2 above)
Comment (bluquar) [reply App.tsx:16]: yep

File: .env (hunk 1 above)
Comment (bluquar) [reply .env:6]: agreed

**Pull request merged**